src/components/RulesSelector.jsx

```diff
@@ -27,19 +29,14 @@ const makeListItems = (filters, onRuleSelected) => (
 const RulesSelector = ({ filters, onRuleSelected }) => {
   const listItems = makeListItems(filters, onRuleSelected);
   return (
-    <div>
+    <div className="rule-selector-container">
```

Comment:
My understanding is that we decided to use objects for styling. I'm not aware of any files that use CSS classes (did a quick search and didn't find any either), with the exception of `Editor.jsx`. However, the reason that component uses them is because it is required for CodeMirror styling.